### DISCUSSION TITLE
Improving Discord Bot Setup to actually work

### DISCUSSION THREAD

**Discussion by Xelfi** [42wim/matterbridge#1907] [General]
Hi there,

I'm just letting you know that this wiki page might be updated (cause it's actually outdated due to new discord bot features):
https://github.com/42wim/matterbridge/wiki/Discord-bot-setup

In fact the section "**Privileged Gateway Intents**" has a missing point:
We also have to **toggle** "Message Content Intent" (which can be pretty obvious) in addition to the "Server Members Intent".
See: http://prntscr.com/NftbbSLjX9it

If the **Message Content Intent** is not enabled, the bridge can no longer get any messages to transfer.

**Comment by qaisjp** [comment]
Issue also created at https://github.com/42wim/matterbridge/issues/1917.

I've resolved the issue, no need for this discussion anymore. Feel free to re-open or follow up on the issue if you have anything to add.
